hybridse/include/base/fe_status.h

```diff
@@ -39,82 +40,81 @@ static inline std::initializer_list<int> __output_literal_args(STREAM& stream,
-    while (true) {                                                                                                    \
-        auto _status = (call);                                                                                        \
-        if (!_status.isOK()) {                                                                                        \
-            std::stringstream _msg;                                                                                   \
-            hybridse::base::__output_literal_args(_msg, ##__VA_ARGS__);                                               \
-            std::stringstream _trace;                                                                                 \
-            hybridse::base::__output_literal_args(_trace, "    (At ", __FILE__, ":", __LINE__, ")");                  \
-            if (_status.trace.size() >= MAX_STATUS_TRACE_SIZE) {                                                      \
-                LOG(WARNING) << "Internal error: " << _status.msg << "\n" << _status.trace;                           \
-            } else {                                                                                                  \
-                if (!_msg.str().empty()) {                                                                            \
-                    _trace << "\n"                                                                                    \
-                           << "    (Caused by) " << _msg.str();                                                       \
-                }                                                                                                     \
-                _trace << "\n" << _status.trace;                                                                      \
-            }                                                                                                         \
-            return hybridse::base::Status(_status.code, _msg.str().empty() ? _status.msg : _msg.str(), _trace.str()); \
-        }                                                                                                             \
-        break;                                                                                                        \
+#define CHECK_STATUS(call, ...)                                         \
+    while (true) {                                                      \
+        auto _status = (call);                                          \
+        if (!_status.isOK()) {                                          \
+            std::stringstream _msg;                                     \
+            hybridse::base::__output_literal_args(_msg, ##__VA_ARGS__); \
+            _status.AddTrace(__FILE__, __LINE__, _msg.str());           \
+            return _status;                                              \
```

Comment:
`msg` in `Status` not changed,  it is different from previous impl.

Reply:
I plan to keep the root message in `msg` and append other messages just into traces.

Reply:
just confirm:
- `root message`: the first message where a error occurred in a sequence

But for end user the last message is more friendly

Reply:
Yep. I plan to keep the first message. Since error messages like "Fail to generate physical plan batch mode" can give users little message.